src/cli/self_update/windows.rs

```diff
@@ -556,10 +568,10 @@ mod tests {
     #[test]
     fn windows_uninstall_removes_semicolon_from_path_suffix() {
         assert_eq!(
-            "foo",
+            wide("foo"),
```

Comment:
does .into() work ? OsString::from(ruststring) should work, and ruststring.into() OsString should reflexively too.